cores/migrations/20240708182646_AddCategoryMarkdown.up.sql

```diff
@@ -0,0 +1,15 @@
+-- Drop the existing primary key constraint
+ALTER TABLE Category DROP CONSTRAINT IF EXISTS category_pkey;
```

Comment:
And this column shouldn't exist in the Category table before this migration right? Not awful to leave it in, but found it a touch confusing since I didn't expect this column to exist when this migration runs lol